### ISSUE TITLE
[feature request] ToC subsection support

### ISSUE THREAD

**Issue by skmatz** [hiroppy/fusuma#181]
## Type

- [ ] bug
- [x] feature request

## Feature Request

I want to display subsections in ToC like below.

```
1. sec1 --------------- 1
  1.1 subsec1 ------- 2
  1.2 subsec2 ------- 3
```

You used to say that it is difficult to customize ToC because the component is provided by webSlides, but I think the feature is in demand, or can I already do that?

**Comment by hiroppy** [comment]
Nope, I want to implement this feature. So we have to implement fusuma's comment before.

https://github.com/hiroppy/fusuma/blob/master/packages/mdx-loader/src/mdxPlugin.js#L41
